### PR TITLE
IDEMPIERE-5414 - Add Help icon to Dashboard Content

### PR DESCRIPTION
**Pull request by PeterTakacs300**
JIRA ticket: https://idempiere.atlassian.net/browse/IDEMPIERE-5414

### CODE REVIEW

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java [hunk 1]
```diff
@@ -335,20 +338,40 @@ private Panel newGadgetPanel(MDashboardPreference dp, MDashboardContent dc) {
...
+			help.setImage(ThemeManager.getThemeResource(IMAGES_CONTEXT_HELP_PNG));
+		Popup popup = new Popup();
+		popup.setSclass("dashboard-content-help");
+		popup.setPopup(popup);
+		Text t = new Text(text);
+		popup.appendChild(t);
+		help.setTooltip(popup);
+		help.addEventListener(Events.ON_CLICK, (Event event) -> {
+			popup.setPage(help.getPage());
+			popup.open(help, "after_start");
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java:373]: need to add LayoutUtils.autoDetachOnClose(popup) call. without which, can lead to memory leak.

Comment (PeterTakacs300) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java:374]: Didn't know that, thank you

**Comment by hengsin** [comment]
hi @PeterTakacs300 , the latest changes works fine but I notice 2 minor issue here:
- alignment between text and the question mark icon seems a bit off. I think the icon is probably 1 or 2 pixel lower than where it should be at.
- should hide the icon when the widget's label and description is the same.

**Comment by PeterTakacs300** [comment]
Hi @hengsin. The problem is, that is seems like the button is centered correctly, and the image within the button is off. Maybe other component would be more correct instead of ToolbarButton? I haven't find another solution yet.

**Comment by hengsin** [comment]
hi @PeterTakacs300, I've try using margin-bottom and it seems to help. Beside toolbarbutton, you can try to use A as well.

**Review by hengsin — APPROVED** [review]
looks good

**Pull request merged**